.delivery/build_cookbook/recipes/provision.rb

```diff
@@ -28,15 +28,6 @@
 # Let delivery-truck do the workflow release promotion
 include_recipe 'delivery-truck::provision'
 
-# Promote the omnibus artifact to current
-expeditor_artifactory workflow_change_project do
-  ref_type 'omnibus.version'
-  ref ::File.read(::File.join(workflow_workspace_repo, 'VERSION')).strip
-  channel 'current'
-  action :promote
-  only_if { workflow_stage?('union') }
-end
```

Comment:
I _think_ without this, there is nothing in the pipeline to promote a build from unstable to current.

BuildKite handles what the ACC smoke test did in the Acceptance stage: the multiple platform builds and the even more multiple platform tests. But if the change in ACC is approved out of Acceptance, this is the resource that performs the promotion from `unstable` to `current` in Union.

Reply:
The Buildkite pipeline which executes the Omnibus build now handles the promotion to current. This promotion is automatically executed after some basic testing of the packages are done. That testing is represented by this script:
https://github.com/chef/supermarket/blob/master/omnibus/omnibus-test.sh

More details on at:
https://expeditor.chef.io/docs/integrations/omnibus/

Reply:
A-ha! I thought _wrong_, then.

Reply:
I'd like to talk about the purpose of Supermarket's ACC pipeline in a world where Expeditor is performing the promotion from unstable to current after a single test run. Right now for a build of Supermarket to land in current, it must

* pass that single local test run (Jenkins' build-test-promote to unstable)
* provision and deploy to ACC's Supermarket Acceptance environment
* pass the smoke tests in Acceptance (I believe those are different tests?)
* get Approved by a human out of Acceptance
* Union's Provision promotes to current

The ACC pipeline does not perform the promotion to stable for release. That is a command to Expeditor. With the smoke tests and human approval removed as a requirement for a build to land in current, what needs are met by what is left of the ACC pipeline activities?

Reply:
For the time being, to serve as an acceptance environment.

Reply:
Another option we could consider is make the promotion to `current` optional in the new Buildkite Omnibus build pipeline and continue to allow the `expeditor_artifactory` resource to promote to `current` in the `union/provision` phase of the workflow pipeline. This would allow things to continue to work int he same way as @robbkidd has come to expect.

Reply:
It's not that I want to preserve old behavior because that's what I expect. I'm wondering how much old behavior can be torn down so it doesn't have to be maintained.

Reply:
Vigorous agreement achieved during video chat with @schisamo, @jeremiahsnapp, and myself.

What remains of the Supermarket ACC pipeline manages Supermarket deployments in the four standard environments (AURD). We'll follow up with folks internally to confirm whether or not those environments are in use or the pipeline itself is a dependency of others.